api/src/main/proto/stellarstation/api/v1/stellarstation.proto

```diff
@@ -112,6 +113,26 @@ service StellarStationService {
   // `ListUpcomingAvailablePasses` should be called again to get a fresh list of reservable passes.
   // The pass must be scheduled at least one hour in advance of `aos_time` or it will be rejected.
   rpc ReservePass (ReservePassRequest) returns (ReservePassResponse);
+
+  // Sets the TLE for a satellite.
```

Comment:
We decided that the latest TLE epoch is always what we use. If we get a newer TLE epoch from another source it will be used by default unless an even newer epoch is set manually again.

Reply:
It's only to provide fresh TLE - the assumption we decided on is the latest epoch of TLE is always the best regardless of source.

It does bring the good point that we should document the fact that the latest TLE is always used for computations and the provided TLE may be overwritten by newer TLE from another source. It does make the word Set not so good though, I like Add best again...

Reply:
Updating the documentation to clarify that latest is always used and reverting back to 'add' is easy enough to do.

Implementing revocation of TLE was what I imagined initially but the 'latest is best' requirement means that it isn't really useful to support such a concept (and it needs a lot of backend work). The frontend will work in a similar way to this API AFAIK.